include/database.py

```diff
@@ -1,18 +1,57 @@
 # database.py
 #
 # This module provides a wrapper for basic database functionality.
+import sys
 import psycopg2
 
+
 def select(connectionString, sql, parameters):
+
     # Open the connection
-    connection = psycopg2.connect(connectionString)
+    try:
+        # connect_timeout for connectionString is 60 in the yml file
+        connection = psycopg2.connect(connectionString, connect_timeout = 5)
+
+    # operational error is raised when parameters passed to connect() are incorrect
+    # or if the server runs out of memory
+
+    except psycopg2.OperationalError as err:
+
+        #print(f' Error Connecting : {err}')
+        sys.stderr.write('Error Connecting: Connection Timed out')
+        # set connection to 'None' in case of error
+        #connection = None
+        sys.exit(1)
+
+    except Exception as exp:
+        sys.stderr.write('Error Connecting')
+
+    # connection is successful
+    #if connection is not None:
+
     cursor = connection.cursor()
```

Comment:
No need to have this outside of the try-catch, remember to think through how the errors might happen. What will happen under normal operations? Connection timeout? Error reported by the database itself?